packages/web/docs/tailwind.config.ts

```diff
@@ -3,20 +3,21 @@ import type { Config } from 'tailwindcss';
 import tailwindcssAnimate from 'tailwindcss-animate';
 import tailwindcssRadix from 'tailwindcss-radix';
 import { fontFamily } from 'tailwindcss/defaultTheme';
+// @ts-expect-error this file has no declarations but it exists
 import { default as flattenColorPalette } from 'tailwindcss/lib/util/flattenColorPalette';
 import plugin from 'tailwindcss/plugin';
 import tailwindTypography from '@tailwindcss/typography';
 import baseConfig from '@theguild/tailwind-config';
 
 const config: Config = {
   ...baseConfig,
+  content: [...baseConfig.content, './mdx-components.js'],
```

Comment:
I also forgot to add `content` dir, can you update shared config? https://github.com/dimaMachina/graphql-eslint/blob/111ab0205d7ba8164b1e8646a8f0009826cf49e9/website/tailwind.config.ts#L6-L7

Reply:
Yup, I'll handle it.